### PR TITLE
Fixes from 2024.08.23

### PR DESCRIPTION
**Pull request by Yukinoroh**
Substitutions:
・U+28EF0: Use "𱣟" instead of "&CDP-8752;"

Corrections:
・U+7476, U+8B21, U+9065: Use "&GT-36228;" instead of "䍃"　　(※Both have ノツ, not 月 - see U+63FA 揺 for reference)
・U+6416, U+7464, U+8B20, U+9059: Use "䍃" instead of "&JX2-7461;"　　(※These really have 䍃)
・U+8083, U+8085: Use "⿻肀" instead of &U-i001+2FF1;聿"
・U+242EE: Use "⿰臣巳" instead of "巸"　　(※巸 contains 𦣝, but what we want is 臣)
・U+23F1C, U+25271, U+259A3, U+25A96: Use "⿻臼乚" instead of "&CDP-88E5;". The reason is "&CDP-88E5;" has 𦥑 (bottom open), while what we need for these is 臼 (bottom closed)).

Simplifications:
(P.S.: I am currently processing the files with a script to find other similar possible simplifications. Hold on!)
・U+8462: Use "盇" instead of "⿱太皿"
・U+28EB3: Use "勇" instead of "⿱甬力"
・U+28EB7: Use "𣅻" instead of "⿱日冋"
・U+28E7C: Use "枈" instead of "⿱比木"
・U+761E, U+22283, U+27490: Use "𢊃" instead of "⿸广⿱夾土"
・U+2B4DE: Use "𬄜" instead of "⿱林㐭"

"?" :
・Replace all "?" (hankaku) with "？" (zenkaku). Some are hankaku, some are zenkaku. Hankaku "?" causes  issues in Linux when processing the files and zenkaku is better to describe kanji anyway.
・U+248D6: Complete decomposition, replace "？" with "豐"
・U+24CBF: Complete decomposition, replace "？" with "𤰱"
・U+250E5: Complete decomposition, replace "？" with "𠄎"
・U+2A5C4: Complete decomposition, replace "？" with "&CDP-89C6;"
・U+38A4: Add tentative decomposition (following pattern of 哉 - need to check the ⿰丿𠃌 part)
・U+2660E, U+24849, U+25B03: Add tentative decomposition
・U+26E46: Add tentative decomposition (following patterns of 𡏳 and 𦿴)
・U+26EF7, U+27065, U+27067: Add tentative decomposition
・U+28DCD: Add tentative decomposition (following info from glyphwiki about central part)
・U+25171: Fix tentative outlook from "⿰" to "⿸"
・U+22013: Fix tentative outlook from "⿰" to "⿻"
・U+22980, U+23AB4, U+25262, U+265B6, U+27DA6, U+27E42, U+285CC, U+294F2, U+296D6: Fix location of unknown element, to ease further work. See 𣪃 for reference.
・U+23C31, U+2487C, U+25057, U+25058, U+25955, U+27E5C: Tentative decomposition is complete; add tab to clearly identify these as "need to check". See 簔 and others for reference.
・U+23E30: Fix tentative decomposition AND add tab to cleary identify it as "need to check".

Question:
・U+2F8FF: Show character instead of a reference to itself
(reverted this change)
Why are some characters in IDS-UCS-Compat-Supplement.txt not printed? Some even reference themselves...

### CODE REVIEW

File: IDS-UCS-Ext-B-3.txt
Comment (chise) [inline IDS-UCS-Ext-B-3.txt:1]: In the structure description of U-000242EE, abstract component <巸> represents abstract component <巸・𭘍>
https://www.chise.org/est/view/character/a.ucs@iwds-1=0x5DF8
and it unifies ⿰臣巳.  cf. UCV-126,163; 120, 121, 122, 123

**Comment by chise** [comment]
It is too large to apply automatic-merge feature, so I will apply your changes by hand.
But if possible, please make new small PRs.
In addition, please avoid adding commits after submitting your PR if possible.

**Comment by Yukinoroh** [comment]
Thank you very much. As I wrote in the description, I am currently processing the files with a bash script to check for possible simplifications. So far after first pass there are 1814 changes, mostly on Extension ranges. But I need a second and possibly third pass (and merge them). How should I submit those changes once they are ready?

**Comment by Yukinoroh** [comment]
Hi! I have the next patch ready. Let me know when you finished merging.

**Pull request closed without merging**